client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -55,9 +58,27 @@ export default React.createClass( {
 		);
 	},
 
+	renderStoreStep() {
+		switch ( abtest( 'signupStoreBenchmarking' ) ) {
+			case 'bluehost':
+			case 'bluehostWithWoo':
+				return (
+					<BluehostStoreStep { ... this.props } onBackClick={ this.handleStoreBackClick } />
+				);
+			case 'siteground':
+				return (
+					<SitegroundStoreStep { ... this.props } onBackClick={ this.handleStoreBackClick } />
+				);
+			default:
+				return (
+					<PressableStoreStep { ... this.props } ref={ this.onPressableStoreStepRef } onBackClick={ this.handleStoreBackClick } />
+				);
+		}
+	},
+
 	render() {
-		let pressableWrapperClassName = classNames( {
-			'design-type-with-store__pressable-wrapper': true,
+		let storeWrapperClassName = classNames( {
```

Comment:
does this need to be a `let`? shouldn't it be constant? if so, let's use `const`